tests/vcr_python_wrapper.py

```diff
@@ -107,11 +107,44 @@ def filter_response(response):
         # headers should be case insensitive, but for some reason they weren't for me
         response['headers'].pop(header.lower(), None)
         response['headers'].pop(header, None)
+    try:
+        json_body = json.loads(response['body']['string'])
+    except json.JSONDecodeError:
+        pass
+    else:
+        if "host" in json_body:
+            json_body['host'] = 'FILTERED'
+        if "account" in json_body:
+            json_body['account'] = 'FILTERED'
+        if "base_dn" in json_body:
+            json_body['base_dn'] = 'FILTERED'
+        if "groups_base" in json_body:
+            json_body['groups_base'] = 'FILTERED'
+        # another for the search result
+        if 'results' in json_body:
+            if "host" in json_body['results'][0]:
+                json_body['results'][0]['host'] = 'FILTERED'
+            if "account" in json_body['results'][0]:
+                json_body['results'][0]['account'] = 'FILTERED'
+            if "base_dn" in json_body['results'][0]:
+                json_body['results'][0]['base_dn'] = 'FILTERED'
+            if "groups_base" in json_body['results'][0]:
+                json_body['results'][0]['groups_base'] = 'FILTERED'
+        response['body']['string'] = json.dumps(json_body).encode()
     return response
 
 
 def filter_request_uri(request):
     request.uri = urlunparse(urlparse(request.uri)._replace(netloc="foreman.example.org"))
+    if request.body is not None:
```

Comment:
Here i think, you should create a new filter function (like `filter_request_manifest`) and call it in the `filter_request` function:
https://github.com/theforeman/foreman-ansible-modules/pull/1083/files#diff-98ccd5edeea7871fb84b2db28d5b369f17bbdfa862faf5ab73618886be95e0a7R157